src/flash/nor/km1m7xx.c

```diff
@@ -0,0 +1,588 @@
+/***************************************************************************
+ * (C) Nuvoton Technology Corporation Japan 2021                           *
+ *   yamaguchi.yoshikazu@nuvoton.com                                       *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>. *
+ ***************************************************************************/
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "imp.h"
+#include <helper/binarybuffer.h>
+#include <target/algorithm.h>
+#include <target/armv7m.h>
+
+#include <target/image.h>
+
+/* Definition for Flash Memory */
+#define	FLASH_SECTOR_SIZE		0x1000
+
+/* Definition for Flash Memory Interface Register */
+#define	FI_BASE_ADDRESS			0x4001C000
+
+#define	FEWEN					0x4001C000
+#define	FEWEN_KEY_CODE			0x2900
+#define	FEWEN_ENABLE			0x004B
+
+#define	SPROSTR					0x4001C018
+#define	SPROSTR_ENABLE			0x00800000
+#define	SPROEND					0x4001C01C
+#define	SPROEND_ENABLE			0x10810000
+
+#define	FWCNT					0x4001C004
+#define	FWCNT_ERASE				0x00000002
+#define	FWCNT_START				0x00000001
+
+#define	FMON					0x4001C008
+#define	FMON_ERROR				0x0000FF00
+#define	FMON_WBUSY				0x00000001
+
+#define	PEADR					0x4001C00C
+
+#define	TIMEOUT_ERASE			100000
+
+struct km1m7xx_flash_bank {
+	int		probed;
+};
+
+/**
+ * @brief	"flash bank" Command
+ * @date	October, 2018
+ * @note	[Usage]	flash bank $_FLASHNAME km1m7xx
+ *					<Address> <size> <ChipWidth> <BusWidth> <Target> <Type>
+ *						<Address>	: Flash memory base address
+ *						<Size>		: Flash memory size
+ *						<ChipWidth>	: Chip width in byte (Not use)
+ *						<BusWidth>	: Bus width in byte (Not use)
+ *						<Target>	: Target device (***.cpu)
+ *						<Type>		: Write contorol type
+ * @param
+ * @return	int			ERROR_OK or the non-zero
+ **/
+FLASH_BANK_COMMAND_HANDLER(km1m7xx_flash_bank_command)
+{
+	struct km1m7xx_flash_bank	*km1m7xx_info;
+
+	km1m7xx_info = malloc(sizeof(struct km1m7xx_flash_bank));
+	bank->driver_priv = km1m7xx_info;
+
+	km1m7xx_info->probed	= 0;
+
+	return(ERROR_OK);
```

Comment:
Remove parentheses unless an expression is returned.
The same issue occurred on the line of 134, 140, 152, 156, 219, 230, 245, 365, 392, 414, 420, 426, 432, 437, 445, 447, 464 , 468, 487.

Reply:
Removed parentheses.
Fixed other similar issues as well. (line 497, 511, 521, 538, 541)